tests/testthat/test-save_e61.R

```diff
@@ -188,145 +211,264 @@ test_that("Does save_data work", {
-    labs_e61(
-      title = "Fairly Lengthy Graph Title With A Lot of Words To Take Up Space",
-      subtitle = "Fairly Lengthy Graph Subtitle With A Lot of Words To Take Up Space",
-      footnotes = "Really long footnote to test that part of the code blah blah blah blah blah blah",
-      sources = c("e61 Institute"),
-      x = NULL, y = "units"
-    )
+  p <- ggplot(data, aes(x, y)) +
+    geom_col()
 
-  rstudioapi::viewer(save_e61(withr::local_tempfile(fileext = ".svg"), graph))
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-y-sml-val.svg", p)))
 
-  # Graph 2
-  graph_2 <- ggplot(graph_data, aes(xcol, y)) +
-    geom_col() +
-    theme_e61() +
-    scale_y_continuous_e61(limits = c(0, 50, 10)) +
-    scale_colour_e61(1) +
-    labs_e61(
-      title = "Fairly Lengthy Graph Title With A Lot of Words To Take Up Space",
-      subtitle = "Fairly Lengthy Graph Subtitle With A Lot of Words To Take Up Space",
-      footnotes = "Really long footnote to test that part of the code blah blah blah blah blah blah",
-      sources = c("e61 Institute"),
-      x = NULL, y = "units"
-    )
+  # Graph with cont-y var with negative values from -20 to 0
+  data <- data.frame(x = factor(1:10), y = runif(10, -20, 0))
 
-  rstudioapi::viewer(save_e61(withr::local_tempfile(fileext = ".svg"), graph_2))
+  p <- ggplot(data, aes(x, y)) +
+    geom_col()
 
-})
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-y-neg-val.svg", p)))
+
+  # Graph with cont-y var with large values from -1000 to +1000
+  data <- data.frame(x = factor(1:10), y = runif(10, -1000, 1000))
+
+  p <- ggplot(data, aes(x, y)) +
+    geom_col()
+
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-y-lg-val.svg", p)))
 
-test_that("Test saving of multi-panel graphs", {
-
-  # This test should be run manually and the results inspected by hand
-  skip("This 'test' is only for interactive purposes")
-
-  # Generate some data
-  graph_data <- data.frame(x = runif(100, 1, 49), y = runif(100, 1, 49),
-                           xcol = 1:100)
-
-  # Graph
-  graph <- ggplot(graph_data, aes(x, y)) +
-    geom_point() +
-    theme_e61() +
-    scale_y_continuous_e61(limits = c(0, 50, 10)) +
-    scale_colour_e61(1) +
-    labs_e61(
-      title = "Graph",
-      footnotes = "Really long footnote to test that part of the code blah blah blah blah blah blah",
-      sources = c("e61 Institute"),
-      x = NULL, y = "units"
+  # Graph with cont x and y vars
+  data <- data.frame(x = runif(10, -1, 1), y = runif(10, -1, 1))
+
+  p <- ggplot(data, aes(x, y)) +
+    geom_point()
+
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-cont-x-y.svg", p)))
+
+  # Graph with discrete x and y vars
+  data <- data.table::CJ(x = factor(1:10), y = factor(1:10))
+  data[, fill := runif(100, 0, 100)]
+
+  p <- ggplot(data, aes(x, y, fill = fill)) +
+    geom_tile()
+
+  expect_snapshot_file(suppressWarnings(save_e61("plot-single-disc-x-y.svg", p)))
```

Comment:
@JackBuckley Debugging suggests this is another case where `check_for_y_var` thinks we have a cont. y-var when its a factor.

Returns `Error: Discrete value supplied to continuous scale`

Reply:
Also fixed this, although geom_tile etc. will look kinda ungly in theme61 unless we make further edits.